values.yaml

```diff
@@ -295,8 +295,15 @@ st2web:
   affinity: {}
   env: {}
   # HTTP_PROXY: http://proxy:1234
+  ## Advanced: To configure st2web to serve HTTPS (port 443) instead of HTTP (port 80)
+  ## provide ssl certs via extra_volumes, and add the ST2WEB_HTTPS env var here:
+  # ST2WEB_HTTPS: 1
```

Comment:
It makes sense to quickly mention in the comments about the alternatives like LoadBalancer or Ingress Controller, which would be a more K8s-native approach to this problem.

Reply:
done